TPL/force-app/main/default/classes/HCCostCaseController.cls

```diff
@@ -25,7 +25,7 @@ public with sharing class HCCostCaseController {
     }
 
     @AuraEnabled
-    public static HCCostCaseDetailWrapper getHealthcareCostsAmbulanceForCase(String caseId, String filterValue, Integer pageSize, Integer pageNumber){
```

Comment:
The controlling attribute sortOrder is now passed to the data access process, and the class has two branches as ascending data and descending data have different requirements.